### PR TITLE
Make Japanese README & Fix Japanese

### PR DESCRIPTION
**Pull request by reindex-ot**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,10 @@
   <a href="/docs/README/zh-tw.md">
   正體中文
   </a>
+  <span> | </span>
+  <a href="/docs/README/ja-jp.md">
```
Comment (FooIbar) [inline README.md:12]: The "-jp" suffix is unnecessary.

File: docs/README/ja-jp.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+軽量で高性能な EhViewer のフォークです。
+
+[Material Design 3](https://m3.material.io/) と[ダイナミックカラー](https://m3.material.io/styles/color/dynamic-color/overview)のサポートを搭載しています。
+
+# ダウンロード
+
+| フレーバー    | 機能                            |
+|-------------|----------------------------------|
+| デフォルト    | Android 8.0 以降、完全対応       |
+| Marshmallow | Android 6.0-7.1、一部対応         |
```
Comment (FooIbar) [inline docs/README/ja-jp.md:76]: This was changed in https://github.com/FooIbar/EhViewer/commit/6fb20fc6c134e88c528a0ce59889c30e2eaaed7b, you may want to check that.
And "default" should not be translated.

Comment (reindex-ot) [reply docs/README/ja-jp.md]: fixed.

File: README.md (hunk 1 above)
Comment (reindex-ot) [reply README.md]: suffix removed.

File: docs/README/ja.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+# 説明
+
+軽量で高性能な EhViewer のフォークです。
+
+[Material Design 3](https://m3.material.io/) と[ダイナミックカラー](https://m3.material.io/styles/color/dynamic-color/overview)のサポートを搭載しています。
+
+# ダウンロード
+
+| フレーバー    | 機能                          |
```
Comment (FooIbar) [inline docs/README/ja.md:73]: The "Minimum Android Version" column is missing.
And "機能" does not matches "Notes".

Comment (reindex-ot) [reply docs/README/ja.md]: okay, fixed,

**Review by FooIbar — APPROVED** [review]
LGTM

**Pull request merged**